### DISCUSSION TITLE
Persistent user choice on prompt

### DISCUSSION THREAD

**Discussion by thoroc** [twpayne/chezmoi#3874] [Q&A] [answered]
As per title, I have tried to search the issues and the discussions without much luck. So how do I get the user prompt to persist on screen when running `chezmoi init` the first time around (or any time there are changes for that matter)?

**Comment by halostatue** [comment, marked as answer]
#3449

You’re looking for the `prompt*Once` functions ([`promptBoolOnce`](https://www.chezmoi.io/reference/templates/init-functions/promptBoolOnce/), [`promptChoiceOnce`](https://www.chezmoi.io/reference/templates/init-functions/promptChoiceOnce/), [`promptIntOnce`](https://www.chezmoi.io/reference/templates/init-functions/promptIntOnce/), and [`promptStringOnce`](https://www.chezmoi.io/reference/templates/init-functions/promptStringOnce/)).
